packages/dashboard/src/App.tsx

```diff
@@ -1,45 +1,41 @@
-import Dashboard from "./Dashboard";
+// Wrappers
+import MantineWrapper from "src/components/wrappers/MantineWrapper";
+import ColorSchemeWrapper from "src/components/wrappers/ColorSchemeWrapper";
+import WagmiWrapper from "src/components/wrappers/WagmiWrapper";
+import { DashProvider } from "src/contexts/DashContext";
+// Router
+import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom";
+// Components
+import Layout from "src/components/composed/Layout";
+import Txs from "src/components/composed/Txs";
+import Palette from "src/components/composed/Palette";
+import MantineGlobal from "src/components/MantineGlobal";
 
-import { Provider, chain, defaultChains, Connector } from "wagmi";
-import { InjectedConnector } from "wagmi/connectors/injected";
-
-import { getDefaultProvider, providers as ethproviders } from "ethers";
-import { getNetwork } from "@ethersproject/providers";
-
-const defaultChain = chain.mainnet;
-
-const getProvider = (_config: { chainId?: number; connector?: Connector }) => {
-  let wProvider = _config.connector?.getProvider(true);
-  console.debug("getProvider", {
-    wProvider,
-    winEth: window.ethereum,
-    _config
-  });
-  let ret: any;
-  if (!wProvider) {
-    ret = getDefaultProvider(getNetwork(_config.chainId ?? defaultChain.id));
-  } else {
-    wProvider
-      .enable()
-      .then((r: any) => console.debug(r))
-      .catch((e: any) => console.error(e));
-    ret = new ethproviders.Web3Provider(wProvider);
-  }
-  console.debug("getProvider.returning", {
-    wProvider,
-    winEth: window.ethereum,
-    ret
-  });
-  return ret;
-};
-
-const connectors = [new InjectedConnector({ chains: defaultChains })];
-
-function App() {
+function App(): JSX.Element {
   return (
-    <Provider connectors={connectors} provider={getProvider}>
-      <Dashboard />
-    </Provider>
+    <div id="app">
+      <ColorSchemeWrapper>
+        <MantineWrapper>
+          <MantineGlobal /> {/* Set global styles */}
+          <WagmiWrapper>
+            <DashProvider>
+              <BrowserRouter>
+                <Routes>
+                  <Route path="/" element={<Layout />}>
+                    <Route index element={<Navigate to="/txs" replace />} />
+                    <Route path="txs" element={<Txs />} />
+                  </Route>
+                  {process.env.NODE_ENV === "development" && (
+                    <Route path="colors" element={<Palette />} />
+                  )}
```

Comment:
This should probably appear in the nav if we're keeping this here (appear in nav only in development, of course)

Reply:
[Nav button added](https://github.com/trufflesuite/truffle/commit/191c6a8572901b9cb3fb8626ba6c80cb7bfac637).